src/vfc_precexp

```diff
@@ -0,0 +1,514 @@
+
+	# run to create profile file
+	run(runPath, output_dir[0], env)
+
+	# get dataset
+	FunctionsFrame, FunctionsIndex, ArgumentsFrame, ArgumentsIndex = getProfile(vfc_profile_file, function_list)
+
+	# set backend
+	set_environment_variable("VFC_BACKENDS", "libinterflop_vprec.so --prec-input-file={} --daz --ftz --mode=full --instrument=operations".format(vfc_config_file), env)
+
+	# Operations
+	print("-------------- Operations --------------")
+	FunctionsFrameCopy = FunctionsFrame.copy()
+	cpt = 1
+	for i in FunctionsIndex:
+		print(cpt, "/", len(FunctionsIndex))
+		search_minimum_operations(	runPath, 
+									cmpPath, 
+									output_dir[0], 
+									output_dir[1],
+									vfc_config_file,
+									env, 
+									ArgumentsFrame,
+									FunctionsFrameCopy,
+									i)
+		cpt += 1
+
+	FunctionsFrameCopy.to_csv("vfc_exp_data/OperationsResults.csv")
+
+	# set backend
+	set_environment_variable("VFC_BACKENDS", "libinterflop_vprec.so --prec-input-file={} --daz --ftz --mode=full --instrument=arguments".format(vfc_config_file), env)
+
+	# Arguments
+	print("-------------- Arguments --------------")
+	ArgumentsFrameCopy = ArgumentsFrame.copy()
+	FunctionsFrameCopy = FunctionsFrame.copy()
+	cpt = 1
+	for i in range(len(FunctionsFrame.index)):
+		tmp = ArgumentsFrameCopy[(ArgumentsFrameCopy['ID'] == FunctionsFrameCopy.at[i, 'ID'])]
+		for j in tmp.index:
+			print(cpt, "/", len(ArgumentsFrameCopy.index))
+			search_minimum_arguments(	runPath, 
+										cmpPath, 
+										output_dir[0], 
+										output_dir[1],
+										vfc_config_file,
+										env, 
+										ArgumentsFrameCopy,
+										FunctionsFrameCopy,
+										j)
+
+			cpt += 1
+
+	ArgumentsFrameCopy.to_csv("vfc_exp_data/ArgumentsResults.csv")
+
+	# set backend
+	set_environment_variable("VFC_BACKENDS", "libinterflop_vprec.so --prec-input-file={} --daz --ftz --mode=full --instrument=all".format(vfc_config_file), env)
+
+	# All
+	print("-------------- All --------------")
+	ArgumentsFrameCopy = ArgumentsFrame.copy()
+	FunctionsFrameCopy = FunctionsFrame.copy()
+	cpt = 1
+	for i in range(len(FunctionsFrameCopy.index)):
+		print(cpt, "/", len(FunctionsFrameCopy.index))
+		for j in tmp.index:
+			search_minimum_arguments(	runPath, 
+										cmpPath, 
+										output_dir[0], 
+										output_dir[1],
+										vfc_config_file,
+										env, 
+										ArgumentsFrameCopy,
+										FunctionsFrameCopy,
+										j)
+		if i in FunctionsIndex:
+			search_minimum_operations(	runPath, 
+										cmpPath, 
+										output_dir[0], 
+										output_dir[1],
+										vfc_config_file,
+										env, 
+										ArgumentsFrameCopy,
+										FunctionsFrameCopy,
+										i)
+		
+		tmp = ArgumentsFrameCopy[(ArgumentsFrameCopy['ID'] == FunctionsFrameCopy.at[i, 'ID'])]
+
+		cpt += 1
+
+	ArgumentsFrameCopy.to_csv("vfc_exp_data/AllArgsResults.csv")
+	FunctionsFrameCopy.to_csv("vfc_exp_data/AllOpsResults.csv")
```

Comment:
Could you check that your python file is PEP8 formatted ? You can use, for example, `autopep8`.

Reply:
it's done